apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -1,5 +1,6 @@
 @use "sass:math";
 @import "phase1-design-system";
+@import 'breakpoints';
```

Comment:
Is `breakpoints` what allows you to do the nth-child stuff?

Reply:
Breakpoints defines variables for the different screen widths, such as tablet and mobile. I was going to add the responsiveness in this PR but opted in making another. Sorry about this ill remove this line for now since it is not being used.

Reply:
Oh gotcha!